### ISSUE TITLE
[FEATURE] Support for custom HTML tags

### ISSUE THREAD

**Issue by terminalkitten** [Riverside-Healthcare/djLint#161]
## Feature Request

Djlint is not indenting any non-standard HTML correctly, i'm using mjml in my django templates:

### Expected output
```
{% block mjml %}
    <mjml>
        <mj-body>
            <mj-container>
                <mj-section>
                    <mj-column>
                        <mj-text>Hello world!</mj-text>
                    </mj-column>
                </mj-section>
            </mj-container>
        </mj-body>
    </mjml>
{% endmjml %}
```

### Current output
```
{% block mjml %}
    <mjml>
    <mj-body>
    <mj-container>
    <mj-section>
    <mj-column>
    <mj-text>Hello world!</mj-text>
    </mj-column>
    </mj-section>
    </mj-container>
    </mj-body>
    </mjml>
{% endmjml %}
```

I've created a PR to allow custom HTML tags set in config, so to support the above situation, one would add to config:

```
custom_html = "mjml,mj-body,mj-container,mj-section,mj-column,mj-text"
```

Only downside for now it's quite cumbersome if you need to support many unknown HTML tags.

-------------

- [X] I've searched the [issues](https://github.com/Riverside-Healthcare/djLint/issues)
- [X] I've read the [docs](https://djlint.com)

**Comment by christopherpickering** [comment]
Nice idea, to make it easier on yourself in the custom config you could use regex, right? `mj-\w+?` would match all the mj- tags right?

So you could config:
```yaml
custom_tags = r"""
   mjml
|  mj-\w+?
```
right?

**Comment by terminalkitten** [comment]
> Nice idea, to make it easier on yourself in the custom config you could use regex, right? `mj-\w+?` would match all the mj- tags right?
> 
> So you could config:
> 
> ```yaml
> custom_tags = r"""
>    mjml
> |  mj-\w+?
> ```
> 
> right?

Yes, I was to fixed on the configfile being TOML format.  I haven’t got time to test this. Tomorrow I can setup a dev machine with this configuration.

**Comment by christopherpickering** [comment]
Thanks, I tested it out and it worked well for me. Cool addition!

**Comment by terminalkitten** [comment]
Thanks for merging this feature in.
